debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresSnapshotChangeEventSource.java

```diff
@@ -269,10 +269,14 @@ protected Optional<String> getSnapshotSelect(RelationalSnapshotContext<PostgresP
     }
 
     protected void setSnapshotTransactionIsolationLevel(boolean isOnDemand) throws SQLException {
-        LOGGER.info("Setting isolation level");
-        String transactionStatement = snapshotter.snapshotTransactionIsolationLevelStatement(slotCreatedInfo, isOnDemand);
-        LOGGER.info("Opening transaction with statement {}", transactionStatement);
-        jdbcConnection.executeWithoutCommitting(transactionStatement);
+        if (connectorConfig.isYbConsistentSnapshotEnabled()) {
```

Comment:
Need to add the YugabyteDBServer.isEnabled() check and handle the else case as well